README.md

```diff
@@ -8,7 +8,7 @@ A layer for adding extra udev rules to your image. Use this for better hardware
 
 Add this to your containerfile:
 
-    COPY --from=ghcr.io/ublue-os/udev-rules etc/udev/rules.d/* /etc/udev/rules.d
+    COPY --from=udev-rules:latest /ublue-os-udev-rules /
```

Comment:
I believe this should be `--from=ghcr.io/ublue-os/udev-rules:latest`

Reply:
I think we should also specify how to use the RPM:

```
COPY --from=udev-rules:latest /ublue-os-udev-rules.noarch.rpm /
RUN rpm -ivh /ublue-os-udev-rules.noarch.rpm
```

Reply:
nice catch